java/test/src/main/java/org/ray/api/test/RuntimeContextTest.java

```diff
@@ -0,0 +1,68 @@
+package org.ray.api.test;
+
+import org.ray.api.Ray;
+import org.ray.api.RayActor;
+import org.ray.api.annotation.RayRemote;
+import org.ray.api.id.UniqueId;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RuntimeContextTest extends BaseTest {
+
+  private static UniqueId DRIVER_ID = UniqueId.randomId();
+  private static String RAYLET_SOCKET_NAME = "/tmp/ray/test/raylet_socket";
+  private static String OBJECT_STORE_SOCKET_NAME = "/tmp/ray/test/object_store_socket";
+
+  @Override
+  public void beforeInitRay() {
+    System.setProperty("ray.driver.id", DRIVER_ID.toString());
+    System.setProperty("ray.raylet.socket-name", RAYLET_SOCKET_NAME);
+    System.setProperty("ray.object-store.socket-name", OBJECT_STORE_SOCKET_NAME);
+  }
+
+  @Test
+  public void testRuntimeContext() {
+    Assert.assertEquals(DRIVER_ID, Ray.getRuntimeContext().getCurrentDriverId());
+    Assert.assertEquals(RAYLET_SOCKET_NAME, Ray.getRuntimeContext().getRayletSocketName());
+    Assert.assertEquals(OBJECT_STORE_SOCKET_NAME,
+        Ray.getRuntimeContext().getObjectStoreSocketName());
+  }
+
+  // test in actor.
+  @RayRemote
+  public static class Info {
+
+    public Info() {
+
+    }
+
+    public UniqueId getCurrentDriverId() {
+      return Ray.getRuntimeContext().getCurrentDriverId();
+    }
```

Comment:
We can merge these methods into one:
```java
public String testRuntimeContext() {
    Assert.assertEquals(DRIVER_ID, Ray.getRuntimeContext().getCurrentDriverId());
    ...
    return "ok";
}
````
And maybe rename the class to `RuntimeContextTester`